common/scala/src/main/scala/whisk/core/entity/ActivationLogs.scala

```diff
@@ -28,7 +28,7 @@ import spray.json.RootJsonFormat
 import spray.json.deserializationError
 import spray.json.pimpAny
 
-protected[core] case class ActivationLogs(private val logs: Vector[String] = Vector()) extends AnyVal {
+protected[core] case class ActivationLogs(val logs: Vector[String] = Vector()) extends AnyVal {
```

Comment:
Without this I get the following warning:
sbt-api: Unhandled type class scala.reflect.internal.Types$MethodType : ($this: whisk.core.entity.ActivationLogs)Vector[String]

And this triggers an error in the dependent projects.

I found these related issues:
http://stackoverflow.com/questions/29720289/warning-about-unhandled-type-class-during-compilation
http://www.scriptscoop2.com/t/3557fd15ab2c/scala-warning-about-unhandled-type-class-during-compilation.html

Seems like a compiler bug.